parts/common/components.json

```diff
@@ -465,23 +465,23 @@
         },
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes/azure-cloud-node-manager",
-          "latestVersion": "v1.28.13-windows-hpc",
-          "previousLatestVersion": "v1.28.13"
+          "latestVersion": "v1.28.14-windows-hpc",
```

Comment:
We want both cloud-node-manager windows hpc and non-windows-hpc since windows-hpc is still rolling out and being monitored, and not in all regions
but here we only have windows-hcp version.

Reply:
cc @Devinwong

Reply:
I've made a PR to fix this: https://github.com/Azure/AgentBaker/pull/5832